### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by beaufortfrancois**

### CODE REVIEW

File: site/en/blog/watch-video-using-picture-in-picture/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Watch video using Picture-in-Picture
 description: >
   Control Picture-in-Picture for video elements on your website.
 date: 2018-10-19
-updated: 2022-07-20
+updated: 2022-08-08
```
Comment (jpmedley) [inline site/en/blog/watch-video-using-picture-in-picture/index.md:7]:
```suggestion
```

Comment (jpmedley) [reply site/en/blog/watch-video-using-picture-in-picture/index.md:7]: https://web.dev/handbook/yaml-front-matter/#updated

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/watch-video-using-picture-in-picture/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply site/en/blog/watch-video-using-picture-in-picture/index.md]: Thanks!

Comment (jpmedley) [reply site/en/blog/watch-video-using-picture-in-picture/index.md]: Can you please commit this.

Comment (beaufortfrancois) [reply site/en/blog/watch-video-using-picture-in-picture/index.md]: I already updated file. See https://github.com/GoogleChrome/developer.chrome.com/pull/3403/files

Comment (jpmedley) [reply site/en/blog/watch-video-using-picture-in-picture/index.md]: Stupid GitHub. 😖

**Review by jpmedley — APPROVED** [review]

**Pull request merged**